app/views/patients/_new_patient.html.erb

```diff
@@ -9,6 +9,11 @@
                        required: true,
                        pattern: "[0-9]{10}|[0-9]{3}\.[0-9]{3}\.[0-9]{4}|[0-9]{3}[-][0-9]{3}[-][0-9]{4}" %>
     </div>
+    <div class="col-2">
+      <%= f.text_field :email,
```

Comment:
Premise here makes sense, but email is pretty sensitive as far as data goes - if we're going to do this I want to make sure it's not available by default, and is opt-in per fund (which we can do by adding a Config along the lines of `Allow Patient Email Field`, default it to false, and let funds turn it to true if they really want it). This is kinda dangerous information to handle.

Reply:
Sounds good, I'll ask specifically why they're looking for email to see if it's really necessary.

Reply:
Just to be clear this is plausible to me, but as a matter of approach whenever we add new fields we have a conversation about whether it's really necessary, what the risks are, whether we need to handle anything special etc. Email is particularly sensitive! so we'll definitely want to have a conversation about it. getting the reasoning/need on record from the fund is definitely appreciated.

Reply:
gotcha. let's target #2688 as your active unit of work for now (since it's reasonably straight forward and doesn't raise data privacy issues). I'll start up an email chain with NH to see if there are end runs around the other fields that don't involve tracking scary stuff.